wgpu-types/src/lib.rs

```diff
@@ -736,24 +736,30 @@ bitflags::bitflags! {
     pub struct DownlevelFlags: u32 {
         /// The device supports compiling and using compute shaders.
         const COMPUTE_SHADERS = 0x0000_0001;
-        /// Supports creating storage images.
-        const STORAGE_IMAGES = 0x0000_0002;
+        /// Supports binding storage buffers and textures to fragment shaders.
+        const FRAGMENT_WRITABLE_STORAGE = 0x0000_0002;
+        /// Supports indirect drawing and dispatching.
+        const INDIRECT_EXECUTION = 0x0000_0004;
+        /// Supports non-zero `base_vertex` parameter to indexed draw calls.
+        const BASE_VERTEX = 0x0000_0008;
```

Comment:
Thought: we should set this to off until we get dx12 and dx11 emulation up and running.

Reply:
Does this also affect base_instance?

Reply:
Actually, no. The `start_instance` is emulated, sort of.
But the problem with base vertex is that it can be negative, and emulating negative base is impossible, it seems to me.
I wrote a comment in the GLES adapter code about this